images/linux/scripts/installers/java-tools.sh

```diff
@@ -7,10 +7,10 @@
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
-DEFAULT_JDK_VERSION=8
-
 set -e
 
+DEFAULT_JDK_VERSION=8
```

Comment:
@alepauly , does it make sense to set Java 11 as default version on Ubuntu 20.04 image? (based on #595)

Reply:
yeah I think we should set 11 as default. @brunoborges would probably like that too. And it's a good preamble to switch it in 18.04 later.

Reply:
Done, 11 set as default for ubuntu20

Reply:
Thanks guys. This is perfect.

Reply:
@brunoborges @alepauly @maxim-lobanov it turned out that Android sdk requires Java8 by default. We've faced the same issue https://stackoverflow.com/questions/46402772/failed-to-install-android-sdk-java-lang-noclassdeffounderror-javax-xml-bind-a

Reply:
You can certainly have JAVA_HOME default to 11, and somewhere in Android SDK point to the 8 install folder.

Reply:
We'll try to figure out possible workaround